### PR TITLE
Updated cs (Czech) translation

### PR DESCRIPTION
**Pull request by rr-vesp**
Commit made via [Stringlate](https://lonamiwebs.github.io/stringlate/)

### CODE REVIEW

File: src/app/src/main/res/values-cs/strings.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <string name="next">Další</string>
     <string name="current_string_id">Stávající ID řetězce:</string>
     <string name="selected_locale">Vybraný jazyk:</string>
-    <string name="welcome">Vítá vás Stringlate! Pro pokračování zadejte název projektu a jeho vlastníka, nebo vložte URL repozitáře z Git.</string>
+    <string name="welcome">Vítá vás Stringlate! Pro pokračování zadejte název projektu a jeho vlastníka, nebo vložte URL repozitáře z GitHubu.</string>
```
Comment (Lonami) [inline src/app/src/main/res/values-cs/strings.xml]: It's actually any Git repository. GitHub is simply an example!

File: src/app/src/main/res/values-cs/strings.xml [hunk 2]
```diff
@@ -74,7 +74,7 @@
     <string name="import_file_success">Import proběhl úspěšně</string>
     <string name="update_applications">Aktualizovat aplikace</string>
     <string name="apps_repo_not_downloaded">Repozitáře aplikací ještě nebyly staženy. Abyste si mohli prohlédnout dostupné aplikace, ťukněte na tlačítko \"%s\" pro stažení rejstříku repozitářů.</string>
-    <string name="sync_failed">Chyba při synchronizaci %s.</string>
+    <string name="sync_failed">Chyba při synchronizaci.</string>
```
Comment (Lonami) [inline src/app/src/main/res/values-cs/strings.xml]: This should keep the %s. It's important.

**Comment by Lonami** [comment]
Thanks :)

File: src/app/src/main/res/values-cs/strings.xml (hunk 2 above)
Comment (rr-vesp) [reply src/app/src/main/res/values-cs/strings.xml]: Ok, thank you.

**Pull request merged**